Sources/StreamChat/Models/Attachments/AnyAttachmentPayload.swift

```diff
@@ -23,6 +41,21 @@ public extension AnyAttachmentPayload {
         )
     }
 
+    /// Creates an instance of `AnyAttachmentPayload` with the URL referencing to a local file.
+    ///
+    /// The resulting attachment will have `ImageAttachmentPayload` if `attachmentType == .image`.
+    /// If the type is different from `.image` the attachment will have payload of `FileAttachmentPayload`
+    /// type.
```

Comment:
This is not true anymore, you have to provide the type explicitly. The type will be what you specify it to be.

Reply:
Hmm, it actually works as the docs say 😄 

- the type is always set to what was provided
- the payload is `ImageAttachmentPayload` if the given type is `.image`, otherwise the `FileAttachmentPayload` is used